src/renderer/store/modules/settings.js

```diff
@@ -195,6 +195,7 @@ const state = {
   hideChannelSubscriptions: false,
   hideCommentLikes: false,
   hideComments: false,
+  hideSpecificChannels: '',
```

Comment:
This is not a bool option, a channel list/collection is what it is
```suggestion
  ChannelsHidden: '',
```